deploy/playbooks/deploy.yml

```diff
@@ -24,3 +24,8 @@
      # this *must* be passed in as an extra-vars option if wanting
      # to report to the production graphite instance
      # graphite_api_key: '1234-asdf-1234'
+
+     # callbacks
+     callback_url: "https://shaman.ceph.com/api/repos/"
+     callback_user: "admin"
```

Comment:
I don't think we want ``callback_user`` or ``callback_key`` defined here anymore with the addition of the ``prod_callbacks.py`` file.